src/mappings/ContractCreation/erc1967/handleContractUpgrade.ts

```diff
@@ -0,0 +1,12 @@
+import { collectionController } from '../../../utils/utilHandlers';
+import { UpgradedLog } from '../../../types/abi-interfaces/Erc1967';
+
+export async function handleContractUpgrade(event: UpgradedLog): Promise<void> {
+  // check interface
+  // Create collection in accordance to interface
+  try {
+    await collectionController(event);
+  } catch (e: any) {
+    if (e?.message === 'Contract does not implement erc165') return;
```

Comment:
Can this be moved into `collectionController`? This handling is repeated a couple of times
```suggestion
    if (e?.message === 'Contract does not implement erc165') return;
    throw e;
```